docs/how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters.md

```diff
@@ -14,14 +14,41 @@ For configuration details, refer to the [official Kubernetes documentation](http
 
 
 <Tabs groupId="k8s-distro">
-<TabItem value="RKE2/K3s" default>
+<TabItem value="RKE2" default>
+
+### Method 1: set the `audit-policy-file` option in `machineGlobalConfig`
+
+Unlike in the upstream RKE2 where the value of the `audit-policy-file` option is expected to be the path to the audit policy file,
+in Rancher you can set the `audit-policy-file` option with the content, and Rancher will convert it to a file and configure RKE2 server options.
```

Comment:
This sounds very similar to the behavior described in https://github.com/rancher/rancher-docs/pull/904, "To make it easier to put files on nodes beforehand, Rancher expects the values of the following options to be the configuration's content, unlike RKE2, where the values should be file paths:" 

Could we mention "configuration's content" here as well to make it clear where the content is? Might seem redundant but "content" on its own feels vague.